src/main/Tree.razor.cs

```diff
@@ -449,6 +454,39 @@ private async Task ScrollToFragment(string elementId)
             }
         }
 
+        //private async void OnTimerInterval(object sender, ElapsedEventArgs e)
```

Comment:
Do we need to keep this commented code fragment? If so, please add a comment why.

Reply:
no we don't need it